libs/styles/iframe.css

```diff
@@ -6,6 +6,13 @@
   padding-bottom: 56.25%;
 }
 
+@media (min-width: 1790px) {
+  .milo-video,
+  .milo-iframe {
+    padding-bottom: 47.25%;
```

Comment:
I almost wonder if adjusting the `min-width` on [line 27](https://github.com/adobecom/milo/blob/9414e399e52834bebcf7a20650f99b3e236f0be6/libs/styles/iframe.css#L28) would be more straight forward and predictable.